### PR TITLE
Improve/workflow

### PR DESCRIPTION
**Pull request by oleksii-sakun**

### CODE REVIEW

File: src/components/CustomInput.jsx [hunk 1]
```diff
@@ -1,30 +1,39 @@
 import React, { useState } from "react";
-import { useDispatch } from "react-redux";
-import { addTask } from "../redux/actions/inputAction";
+import { Button, Input } from "semantic-ui-react";
+import { createTaskRequest } from "../api";
+import "semantic-ui-css/semantic.min.css";
 
 export default function CustomInput(props) {
```
Comment (ArchieMollaev) [inline src/components/CustomInput.jsx:6]: Name of component should be self-explanatory

File: src/components/CustomInput.jsx [hunk 2]
```diff
@@ -1,30 +1,39 @@
...
-      <button className="add_task_btn" onClick={handleButtonCreateTask}>
-        Add
-      </button>
+        value={inputValue}
+      ></Input>
+      <Button
+        positive
+        className="add_task_btn"
+        onClick={() =>
+          handleActionForAddTaskButton(inputValue, Number(props.column))
```
Comment (ArchieMollaev) [inline src/components/CustomInput.jsx:32]: all ids are number by default. Remove Number

Comment (ArchieMollaev) [inline src/components/CustomInput.jsx:32]: all ids are number by default. Remove Number

Comment (ArchieMollaev) [inline src/components/CustomInput.jsx:32]: all ids are number by default. Remove Number

File: src/components/CustomInput.jsx (hunk 1 above)
Comment (oleksii-sakun) [reply src/components/CustomInput.jsx:6]: fixed

File: src/components/CustomInput.jsx (hunk 2 above)
Comment (oleksii-sakun) [reply src/components/CustomInput.jsx:32]: fixed

**Pull request merged**